classes/controller/AdminController.php

```diff
@@ -1826,11 +1826,19 @@ public function display()
             $this->context->smarty->assign(
                 array(
                     'page_header_toolbar' => $this->context->smarty->fetch($page_header_toolbar),
-                    'modal_module_list' => $this->context->smarty->fetch($modal_module_list),
                 )
             );
+            if (!empty($modal_module_list)) {
```

Comment:
With the ternary condition, $modal_module_list is never empty

Reply:
yes an empty string returns true, that's the point of the test

Reply:
Oh my bad, miss reading the diff and thinking the ternary was:
```php
file_exists($module_list_dir . 'modal.tpl') ? $module_list_dir . 'modal.tpl' : 'modal.tpl';
```

Reply:
yes that was the previous ternary, but with the new template it no longer has the wanted behavior and instead of adding a modal for available modules (which does not exist in the new theme) it resulted in including the addon login modal in two different locations